Packs/Jira/Integrations/JiraV2/JiraV2.yml

```diff
@@ -104,6 +105,11 @@ configuration:
   name: fetch_attachments
   required: false
   type: 8
+- display: API token (Deprecated)
```

Comment:
should we set this param to `hidden: true`?
it will not harm the current integration instances, and will be ease new ones.

Reply:
if we have hidden then we can

Reply:
good idea, thanks @bakatzir